examples/soap_api_usecases/convert_lead/convert_lead.bal

```diff
@@ -16,14 +16,13 @@
 
 import ballerinax/salesforce.soap;
 import ballerina/log;
-import ballerina/os;
 
 // Create Salesforce client configuration by reading from environemnt.
```

Comment:
```suggestion
// Create Salesforce client configuration by reading from environment.
```

Reply:
check for other places

Reply:
Updated in [Update code according to comments](https://github.com/ballerina-platform/module-ballerinax-sfdc/pull/322/commits/637bff31ea6fdcb76c54d48b8f7054b644e6d85a)